RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, and where we’d like to grow in the future. 
+
+
+## Core Strategy Statement  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## Who we are
+
+We are a community of technical folk. Writers, developers, editors and more. We are passionate about technical writing, documentation and open source.
+
+
+### Our Audience
+
+Our primary audience includes:
+
+* Users of our products
+* Members of our community
+* People seeking to learn more about the technical writing domain
```

Comment:
That might get into roles and content types. I know there were some discussions when I was attending meetings about how some content may be internally for community members or aspirants (but still content on the website), so that's one user segment.

Reply:
I will leave this section as-is but have captured these comments in the feedback section of the RFC. I think it's an important conversation and one I wrangled with when putting the core strategy together. Let's totally talk and iterate!